packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -118,14 +120,15 @@ export class DataSyncPlugin extends GraphbackPlugin {
   protected addDataSyncFieldsToModel(schemaComposer: SchemaComposer<any>, model: ModelDefinition) {
     const name = model.graphqlType.name;
     const modelTC = schemaComposer.getOTC(name);
+    const TimestampSTC = schemaComposer.getSTC(GraphbackTimestamp.name);
```

Comment:
There is no need for this, you can just refer to the scalar directly (this is getting the type from the type name, but you already have the type).

Reply:
Hmm, I tried doing that before(and now) as well, it throws when GraphbackTimestamp is already present and used somewhere in the schema:
![image](https://user-images.githubusercontent.com/20183094/89439171-6a1b5180-d767-11ea-8b43-13bd4ce96bfc.png)
wdyt

Reply:
My bad, thanks for checking.

You could still access the type directly with SchemaComposer:

```ts
const TimestampScalar = schemaComposer.types.get(GraphbackTimestamp.name);
```